.github/workflows/php.yml

```diff
@@ -0,0 +1,45 @@
+name: OrangeHRM Build (Ubuntu)
+
+on: [push, pull_request]
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/checkout@v2
+
+            -   name: Validate composer.json and composer.lock
+                run: composer validate -d ./symfony/lib
+
+            -   name: Install dependencies
+                run: composer install -d ./symfony/lib
+
+            -   name: Dumps the autoloader
+                run: composer dump-autoload -o -d ./symfony/lib
+
+            -   uses: getong/mariadb-action@v1.1
+                with:
+                    host port: 3306
+                    container port: 3306
+                    character set server: 'utf8mb4'
+                    collation server: 'utf8mb4_general_ci'
+                    mariadb version: '10.2.31'
+                    mysql database: 'orangehrm'
+                    mysql root password: ${{ secrets.RootPassword }}
+                    mysql user: 'developer'
+                    mysql password: ${{ secrets.DatabasePassword }}
+
+            -   uses: shivammathur/setup-php@v2
```

Comment:
can't we get via php official repo ?

Reply:
Do you have any reference? I used [this](https://github.com/marketplace/actions/setup-php-action) one.